.circleci/config.yml

```diff
@@ -39,6 +39,15 @@ jobs:
           # XSLTTransformation needs libxml2.
           name: Install libxml2
           command: sudo apt-get update && sudo apt install -y libxml2-dev libxslt1-dev liblzma-dev zlib1g-dev
+      - run:
+          # DataWeaveTransformation needs dw cli
+          name: Install DataWeave CLI
+          command: |
+            mkdir /tmp/dw
+            curl -sSLO https://github.com/mulesoft-labs/data-weave-cli/releases/download/v$DW_VERSION/dw-$DW_VERSION-Linux
+            unzip -p dw-$DW_VERSION-Linux 'bin/dw' > dw && chmod +x dw && mv dw $(go env GOPATH)/bin/
+          environment:
+            DW_VERSION: 1.0.19
```

Comment:
Could you please provide explanation as to why we need that step? Nothing in the CI seems to be leveraging this CLI after this installation step.

Did you mean to install `dw` in a Dockerfile at release time maybe?

Reply:
My bad, I didn't see the adapter test!